pytorch_lightning/plugins/plugin_connector.py

```diff
@@ -94,3 +100,80 @@ def __attach_cluster(self, limit=1):
 
                 # set the cluster
                 self.cloud_environment = plugin
+
+    def _str_to_lightning_custom_plugins(self, plugins: Union[str, list]):
```

Comment:
do we need this extra when it just calls the very same method? or better name...
```suggestion
    def _convert_str_custom_plugins(self, plugins: Union[str, list]):
```